app/src/main/res/layout/featured_search_result_view.xml

```diff
@@ -15,87 +16,101 @@
-          android:layout_weight="0"
-          android:textStyle="bold" />
+          android:maxLines="2"
+          android:ellipsize="end"
+          tools:text="Gravity: The Weighted Blanket for Sleep, Stress and Anxiety" />
 
-        <TextView
-          android:id="@+id/project_stats_text_view_percent_complete_string"
-          style="@style/ActivityFeedCard"
+        <LinearLayout
+          android:id="@+id/featured_search_result_stats"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_weight="0"
-          android:maxLines="1"
-          android:textColor="@color/black_alpha_50" />
+          android:layout_marginTop="@dimen/grid_new_2"
+          android:layout_marginStart="@dimen/grid_new_2"
+          android:layout_marginLeft="@dimen/grid_new_2"
+          android:layout_marginRight="@dimen/grid_new_2"
+          android:layout_marginEnd="@dimen/grid_new_2"
+          android:layout_marginBottom="@dimen/grid_new_1"
+          android:orientation="horizontal">
 
-        <TextView
-          android:id="@+id/project_stats_text_view_days_to_go_data"
-          style="@style/ActivityFeedCard"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_weight="0"
-          android:maxLines="1"
-          android:textColor="@color/gray_black"
-          android:textStyle="bold" />
+          <TextView
+            android:id="@+id/search_result_percent_funded_text_view"
+            style="@style/BodyPrimaryMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/green"
+            android:layout_marginEnd="@dimen/grid_new_1_half"
+            android:layout_marginRight="@dimen/grid_new_1_half"
+            tools:text="8,000%"/>
+
+          <TextView
+            android:id="@+id/search_result_funded_text_view"
+            style="@style/BodySecondary"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/grid_new_2"
+            android:layout_marginRight="@dimen/grid_new_2"
+            tools:text="funded"/>
+
+          <TextView
+            android:id="@+id/search_result_deadline_countdown_text_view"
+            style="@style/BodyPrimaryMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/grid_new_1_half"
+            android:layout_marginRight="@dimen/grid_new_1_half"
+            tools:text="25"/>
+
+          <TextView
+            android:id="@+id/search_result_deadline_unit_text_view"
+            style="@style/BodySecondary"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="days to go"/>
+
+        </LinearLayout>
```

Comment:
This should really be one text view with html.  Since you are rewriting the UI anyway, it makes sense to do it properly.  If someone has a small screen and a large font, you will get some very weird UI with this chained textview method.  Also it doesn't localize properly and word order is fixed.